Source/Core/DolphinQt/Config/GameConfigEdit.cpp

```diff
@@ -207,31 +207,154 @@ void GameConfigEdit::AddMenubarOptions()
-                  QStringLiteral("Video_Hacks"), QStringLiteral("EFBToTextureEnable"));
+    auto* console_menubar = m_menu->addMenu(tr("Console"));
 
-    AddBoolOption(video_menubar, tr("Store XFB Copies to Texture Only"),
-                  QStringLiteral("Video_Hacks"), QStringLiteral("XFBToTextureEnable"));
+    AddBoolOption(console_menubar, tr("Skip Main Menu (GC)"), QStringLiteral("Core"),
+                  QStringLiteral("SkipIPL"));
+
+    AddBoolOption(console_menubar, tr("Use PAL60 Mode (Wii)"), QStringLiteral("Core"),
+                  QStringLiteral("PAL60"));
+
+    AddBoolOption(console_menubar, tr("Aspect Ratio (Wii)"), QStringLiteral("Core"),
+                  QStringLiteral("Widescreen"));
+
+    AddBoolOption(console_menubar, tr("Insert SD Card (Wii)"), QStringLiteral("Core"),
+                  QStringLiteral("WiiSDCard"));
+
+    AddBoolOption(console_menubar, tr("Allow Writes to SD Card (Wii)"), QStringLiteral("Wii"),
+                  QStringLiteral("WiiSDCardAllowWrites"));
 
     {
-      auto* texture_cache = video_menubar->addMenu(tr("Texture Cache"));
-      texture_cache->addAction(tr("Safe"), this, [this] {
-        SetOption(QStringLiteral("Video_Settings"), QStringLiteral("SafeTextureCacheColorSamples"),
-                  QStringLiteral("0"));
-      });
-      texture_cache->addAction(tr("Medium"), this, [this] {
-        SetOption(QStringLiteral("Video_Settings"), QStringLiteral("SafeTextureCacheColorSamples"),
-                  QStringLiteral("512"));
-      });
-      texture_cache->addAction(tr("Fast"), this, [this] {
-        SetOption(QStringLiteral("Video_Settings"), QStringLiteral("SafeTextureCacheColorSamples"),
-                  QStringLiteral("128"));
+      auto* cpu_engine = core_menubar->addMenu(tr("CPU Emulator Engine"));
+      cpu_engine->addAction(tr("JIT x64"), this, [this] {
+        SetOption(QStringLiteral("Core"), QStringLiteral("CPUCore"), QStringLiteral("1"));
+      });
+      cpu_engine->addAction(tr("Cached Interpreter"), this, [this] {
+        SetOption(QStringLiteral("Core"), QStringLiteral("CPUCore"), QStringLiteral("5"));
+      });
+      cpu_engine->addAction(tr("Interpreter"), this, [this] {
+        SetOption(QStringLiteral("Core"), QStringLiteral("CPUCore"), QStringLiteral("0"));
+      });
+      auto* emulation_speed = core_menubar->addMenu(tr("Emulation Speed"));
+      emulation_speed->addAction(tr("Unlimited"), this, [this] {
+        SetOption(QStringLiteral("Core"), QStringLiteral("EmulationSpeed"),
+                  QStringLiteral("0.000000000"));
+      });
+      emulation_speed->addAction(tr("10%"), this, [this] {
+        SetOption(QStringLiteral("Core"), QStringLiteral("EmulationSpeed"),
+                  QStringLiteral("0.100000001"));
+      });
+      emulation_speed->addAction(tr("20%"), this, [this] {
+        SetOption(QStringLiteral("Core"), QStringLiteral("EmulationSpeed"),
+                  QStringLiteral("0.200000003"));
```

Comment:
There's no need to keep float rounding quirks in here. Just do 0.0, 0.1, 0.2 and so on.